### PR TITLE
addition after Ian's input

### PR DESCRIPTION
**Pull request by AshVic**

### CODE REVIEW

File: docs/command-reference.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ sidebar_label: Command reference
 
 ## System service commands
 
-The service scripts are used to control operation of each individual service.
+The service scripts are used to control operation of each individual service. In most supported Linux distributions the following commands can be used to manage LiveData Migrator, Hive Migrator, and LiveData UI processes.
```
Comment (stepgerner) [inline docs/command-reference.md:10]: Maybe just me, but I think a comma after "distributions" would fit, and the sentence should end on a ":"

File: docs/configure-source.md [hunk 2]
```diff
@@ -338,7 +338,7 @@ Manage the source filesystem with the following commands:
 
 :::note
 To update existing filesystems, first stop all migrations associated with them.
-Then go to your terminal and run the command `service livedata-migrator restart` for your updates to take effect.
+After saving updates to your configuration, you'll need to restart the LiveData Migrator service for your updates to take effect. In most supported Linux distributions, run the command `service livedata-migrator restart`.
```
Comment (stepgerner) [inline docs/configure-source.md:341]: I suggest rephrasing the second sentence to "Use `service livedata-migrator restart` to do this in most supported Linux distributions."

File: docs/configure-target.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ Update a target filesystem's configuration in its **Filesystem Configuration** p
 
 :::note
 To update existing filesystems, first stop all migrations associated with them.
-Then go to your terminal and run the command `service livedata-migrator restart` for your updates to take effect.
+After saving updates to your configuration, you'll need to restart the LiveData Migrator service for your updates to take effect. In most supported Linux distributions, run the command `service livedata-migrator restart`.
```
Comment (stepgerner) [inline docs/configure-target.md:38]: Same as above

File: docs/command-reference.md (hunk 1 above)
Comment (AshVic) [reply docs/command-reference.md:10]: agreed

**Pull request merged**